girder/api/sftp.py

```diff
@@ -261,32 +262,25 @@ def shutdown_request(self, request):
         pass
 
 
-def _main():  # pragma: no cover
+@click.command(name='sftpd', short_help='Run the Girder SFTP service.')
+@click.option('-i', '--identity-file', help='path to identity (private key) file')
+@click.option('-p', '--port', default=DEFAULT_PORT, type=int)
```

Comment:
Might as well take this chance to add `help` to the `--port` and `--host` options. It should be fine to copy the strings from `girder serve`.